xml/System.Reflection/MetadataAssemblyResolver.xml

```diff
@@ -54,11 +54,30 @@
         <Parameter Name="assemblyName" Type="System.Reflection.AssemblyName" />
       </Parameters>
       <Docs>
-        <param name="context">To be added.</param>
-        <param name="assemblyName">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="context">The metadata load context object that represents the closed universe of Type objects loaded for inspection-only purposes.</param>
+        <param name="assemblyName">The name of the assembly.</param>
+        <summary>Returns an assembly from a given assembly name.</summary>
+        <returns>The assembly for the provided <paramref name="assemblyName" />.</returns>
+        <remarks>
+          <format type="text/markdown">
+  <![CDATA[
+
+## Remarks
+
+Binding an assembly name to an assembly occurs when <xref:System.Reflection.MetadataLoadContext.LoadAssemblyByName> is called or when a Type from one assembly has a dependency on another assembly. It should use <xref:System.Reflection.MetadataLoadContext.LoadFromStream>, <xref:System.Reflection.LoadFromAssemblyPath> or <xref:System.Reflection.LoadFromByteArray> to load the requested assembly and return it.
```

Comment:
```suggestion
Binding an assembly name to an assembly occurs when <xref:System.Reflection.MetadataLoadContext.LoadFromAssemblyName%2A> is called or when a type from one assembly has a dependency on another assembly. It should use <xref:System.Reflection.MetadataLoadContext.LoadFromStream%2A>, <xref:System.Reflection.MetadataLoadContext.LoadFromAssemblyPath> or <xref:System.Reflection.MetadataLoadContext.LoadFromByteArray%2A> to load the requested assembly and return it.
```
"It" at the beginning of the second sentence, though, has no referent. Perhaps "The handler that overrides `\Resolve\`...?

Reply:
I got it from here:
https://github.com/dotnet/corefx/blob/775347f642ae10badf82bcd0352c9f316093cd6f/src/System.Reflection.MetadataLoadContext/src/System/Reflection/MetadataAssemblyResolver.cs#L17

Judging by the fact that it was originally inside the summary for the Resolve method, it sounds to me like "it" refers to the Resolve method, but I'm not really sure. @steveharter @GrabYourPitchforks what do you think? Can you chime in?

Reply:
```suggestion
Binding an assembly name to an assembly occurs when <xref:System.Reflection.MetadataLoadContext.LoadFromAssemblyName%2A> is called or when a type from one assembly has a dependency on another assembly. The handles that overrides Resolve should use <xref:System.Reflection.MetadataLoadContext.LoadFromStream%2A>, <xref:System.Reflection.MetadataLoadContext.LoadFromAssemblyPath> or <xref:System.Reflection.MetadataLoadContext.LoadFromByteArray%2A> to load the requested assembly and return it.
```